### PR TITLE
Add documentation for build java-object

### PR DESCRIPTION
**Pull request by tdrozdovsky**
Signed-off-by: Taras Drozdovskyi <t.drozdovsky@samsung.com>

# Description

Due to the lack of a description of how to set up the environment for building a java-object, an error arose when building for the android platform. This PR only covers setting up the environment for a successful java-object build. 
**An example how to edge-orchestration works for the android platform will be prepared later in another branch (due to the complexity of this task).**

## Type of change

- [x] Extending documentation

# How Has This Been Tested?

How to test is described in the edge-home-orchestration-go/doc/platforms/x86_64_linux/x86_64_android.md

```
Example
1. Describe the reproduction procedures freely.
2. Or list up the test description like :
  - [ ] Unittest
  - [ ] Execution of Container
  - [x] Execution on top of Android
```

**Test Configuration**:
* Firmware version: Ubuntu 16.04, etc.)
* Hardware: x86-64
* Toolchain: Recommended versions
* Edge Orchestration Release: Baobab

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings

### CODE REVIEW

File: doc/platforms/x86_64_linux/x86_64_android.md [hunk 1]
```diff
@@ -0,0 +1,113 @@
+# Edge Orchestration on x86_64 (TypeExec: android)
+
+## Prerequisites:
+- Android SDK
+1. Download [Android SDK](https://developer.android.com/studio/index.html#command-tools), then extract the downloaded package.
+2. Wrapping `tools` directory inside `cmdline-tools`
+3. Execute folowing commands:
```
Comment (MoonkiHong) [inline doc/platforms/x86_64_linux/x86_64_android.md:7]:
```suggestion
3. Execute following commands: 
```

File: doc/platforms/x86_64_linux/x86_64_android.md [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+3. Execute folowing commands: 
+```
+$ ./sdkmanager "platform-tools" "platforms;android-28"
+$ export ANDROID_SDK=<path to Android SDK directory>
+$ export ANDROID_HOME=<path to Android SDK directory>
+$ export PATH=$PATH:$ANDROID_SDK/cmdline-tools/tools:$ANDROID_SDK/platform-tools
+```
+- Android NDK
+1. Download the latest version of [Android NDK](https://developer.android.com/ndk/downloads), then extract the downloaded file.
+2. Execute folowing commands:
```
Comment (MoonkiHong) [inline doc/platforms/x86_64_linux/x86_64_android.md:16]:
```suggestion
2. Execute following commands:
```

Comment (t25kim) [inline doc/platforms/x86_64_linux/x86_64_android.md:16]: I don't know why your guide didn't work for me but I made it as below.
1. Go to cmdline-tools/tools/bin and Execute following commands:
```
$ ./sdkmanager --install ndk-bundle
```

File: doc/platforms/x86_64_linux/x86_64_android.md [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+drwxrwxr-x 2 virtual-pc virtual-pc    4096 лип 22 00:09 .
+drwxrwxr-x 3 virtual-pc virtual-pc    4096 лип 22 00:09 ..
+-rw-rw-r-- 1 virtual-pc virtual-pc 6444465 лип 22 00:09 liborchestration.aar
+-rw-rw-r-- 1 virtual-pc virtual-pc    9996 лип 22 00:09 liborchestration-sources.jar
+
+
+**********************************
+ Edge-orchestration Archive 
+**********************************
+tree /home/virtual-pc/projects/edge-home-orchestration-go/src/interfaces/capi/output
```
Comment (t25kim) [inline doc/platforms/x86_64_linux/x86_64_android.md:85]: Someone may need 'sudo apt-get install tree'

File: doc/platforms/x86_64_linux/x86_64_android.md (hunk 2 above)
Comment (tdrozdovsky) [reply doc/platforms/x86_64_linux/x86_64_android.md]: Done. Thank you!

File: doc/platforms/x86_64_linux/x86_64_android.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (tdrozdovsky) [reply doc/platforms/x86_64_linux/x86_64_android.md]: Done. Thank you!

File: doc/platforms/x86_64_linux/x86_64_android.md (hunk 2 above)
Comment (tdrozdovsky) [reply doc/platforms/x86_64_linux/x86_64_android.md]: Done. Thank you!

File: doc/platforms/x86_64_linux/x86_64_android.md (hunk 3 above)
Comment (tdrozdovsky) [reply doc/platforms/x86_64_linux/x86_64_android.md:84]: Done. Thank you!

File: doc/platforms/x86_64_linux/x86_64_android.md (hunk 2 above)
Comment (tdrozdovsky) [reply doc/platforms/x86_64_linux/x86_64_android.md]: Done. Thank you!

File: doc/platforms/x86_64_linux/x86_64_android.md (hunk 1 above)
Comment (tdrozdovsky) [reply doc/platforms/x86_64_linux/x86_64_android.md]: Done. Thank you!

**Comment by MoonkiHong** [comment]
@tdrozdovsky Build succeeded. Please add the "Co-authored-by: <t25.kim@samsung.com>" into your recent commit.

**Comment by tdrozdovsky** [comment]
@MoonkiHong I've added the t25kim as co-author.

**Comment by MoonkiHong** [comment]
> @MoonkiHong I've added t25kim as co-author.

@tdrozdovsky Please rectify one of the co-authors, which is you. You are the original contributor/author. https://github.com/lf-edge/edge-home-orchestration-go/pull/114/commits/971d2aa8cf253d9478466b38fc885028dc87b266

**Comment by tdrozdovsky** [comment]
@MoonkiHong Done)

**Pull request merged**